plugins/jobs/src/js/pages/JobDetailPage.js

```diff
@@ -0,0 +1,193 @@
+import classNames from "classnames";
+import prettycron from "prettycron";
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+import { routerShape } from "react-router";
+import mixin from "reactjs-mixin";
+
+import JobFormModalContainer from "#PLUGINS/jobs/src/js/JobFormModalContainer";
+import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
+
+import Icon from "#SRC/js/components/Icon";
+import Page from "#SRC/js/components/Page";
+import TimeAgo from "#SRC/js/components/TimeAgo";
+import TabsMixin from "#SRC/js/mixins/TabsMixin";
+import StringUtil from "#SRC/js/utils/StringUtil";
+import Job from "#SRC/js/structs/Job";
```

Comment:
Mediator is passing in a new job type, but this is still using the old struct? I'm confused 🤔

Reply:
yes, the mediator is using the graphql resolved json for the most part, except for the `JobFormModalContainer`, which is part of DCOS-37611 AFAIK. It uses the json property on the graphql output to pass a valid version over to the container (it doesn't need the runs and things like that that are filtered out by the json representation)